string node_name
geometry_msgs/Pose new_pose
---
bool success
